bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/items/RollershutterItemTest.java

```diff
@@ -49,4 +57,43 @@ public void setState_stateDecimal050_returnPercent50() {
         assertEquals(new PercentType(50), sut.getState());
     }
 
+    @Test
+    public void setState_stateOpen_returnPercent100() {
+        RollershutterItem sut = new RollershutterItem("Test");
+        State state = OpenClosedType.OPEN;
+        sut.setState(state);
```

Comment:
imho returning null is even the stronger answer when it comes to detecting a failed conversion.